### PR TITLE
[ios] Removed crashlytics

### PR DESCRIPTION
**Pull request by biodranik**
TODO: Remove crash* strings when remove crashlytics on Android.

### CODE REVIEW

File: storage/diff_scheme/diff_scheme_loader.cpp [hunk 1]
```diff
@@ -105,7 +105,7 @@ NameDiffInfoMap DeserializeResponse(string const & response, LocalMapsInfo::Name
 
 NameDiffInfoMap Load(LocalMapsInfo const & info)
 {
-  if (info.m_localMaps.empty())
+  if (info.m_localMaps.empty() || std::strlen(DIFF_LIST_URL) == 0)
```
Comment (vng) [inline storage/diff_scheme/diff_scheme_loader.cpp:108]: I'd prefer DIFF_LIST_URL[0] == 0 here

File: iphone/Maps/Maps.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5222,60 +5212,6 @@
 			shellPath = /bin/sh;
 			shellScript = "#!/bin/bash\necho \"[Firebase Crashlytics] start\"\n\nLOWERED_CONFIG=`echo $CONFIGURATION | tr [A-Z] [a-z]`\n\nPLIST_PATH=\"${PROJECT_DIR}/${BUNDLE_IDENTIFIER}/GoogleService-Info.plist\"\nDSYM_PATH=\"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}\"\nBUILD_APP_DIR=\"${BUILT_PRODUCTS_DIR}/${FULL_PRODUCT_NAME}\"\n\necho \"[Firebase Crashlytics] dsyms: ${DSYM_PATH}\"\necho \"[Firebase Crashlytics] plist: ${PLIST_PATH}\"\n\nif [[ $LOWERED_CONFIG == *release* ]]; then\n  if [ ! -f ${PLIST_PATH} ]; then\n    echo \"[Firebase Crashlytics] Google plist not found!\"\n    exit 1\n  fi\n  cp ${PLIST_PATH} $BUILD_APP_DIR/GoogleService-Info.plist\n  \n  #TODO: Do not upload symbols each build (too large). Do it manually or move to production config. \n  #${PODS_ROOT}/FirebaseCrashlytics/upload-symbols -gsp ${PLIST_PATH} -p ios ${DSYM_PATH}\n  \n  echo \"[Firebase Crashlytics] done\"\nfi\n";
```
Comment (vng) [inline iphone/Maps/Maps.xcodeproj/project.pbxproj:5213]: This additional shell is also not needed

File: iphone/Maps/Maps.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -4963,7 +4953,7 @@
 				"en-GB",
 				el,
 			);
-			mainGroup = 29B97314FDCFA39411CA2CEA /* Maps */;
+			mainGroup = 29B97314FDCFA39411CA2CEA;
```
Comment (vng) [inline iphone/Maps/Maps.xcodeproj/project.pbxproj:4956]: Why comment was removed?

File: iphone/Maps/Maps.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2266,7 +2259,7 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
-		29B97314FDCFA39411CA2CEA /* Maps */ = {
+		29B97314FDCFA39411CA2CEA = {
```
Comment (vng) [inline iphone/Maps/Maps.xcodeproj/project.pbxproj:2262]: Why comment was removed?

File: iphone/Maps/Maps.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -2218,8 +2213,6 @@
 		044715AD432F538B6F731670 /* Pods */ = {
 			isa = PBXGroup;
 			children = (
-				6713C7E64B4DBC46FEB37991 /* Pods-OMaps.debug.xcconfig */,
-				2ED44CE4F4879D8E8BC4A9E0 /* Pods-OMaps.release.xcconfig */,
```
Comment (vng) [inline iphone/Maps/Maps.xcodeproj/project.pbxproj:2222]: All group is not needed?

**Review by vng — CHANGES_REQUESTED** [review]

**Review by rtsisyk — APPROVED** [review]
Please don't mix unrelated changes in the same PR the next time.

**Comment by biodranik** [comment]
Fixed. Thanks!

**Pull request merged**